include/maidsafe/routing/node_info.h

```diff
@@ -34,43 +34,36 @@ namespace routing {
 struct NodeInfo {
   using PublicPmid = passport::PublicPmid;
 
-  NodeInfo(NodeId id, PublicPmid dht_fob) : id(id), dht_fob(std::move(dht_fob)) {}
-  NodeInfo() = default;
+  NodeInfo(NodeId id_in, PublicPmid dht_fob_in, bool connected_in)
+      : id(id_in), dht_fob(std::move(dht_fob_in)), connected(connected_in) {}
+
+  NodeInfo() : id(), dht_fob(), connected(false) {}
+
   NodeInfo(const NodeInfo&) = default;
 
   NodeInfo(NodeInfo&& other) MAIDSAFE_NOEXCEPT : id(std::move(other.id)),
-                                                 dht_fob(std::move(other.dht_fob)) {}
+                                                 dht_fob(std::move(other.dht_fob)),
+                                                 connected(std::move(other.connected)) {}
+
   NodeInfo& operator=(const NodeInfo&) = default;
 
   NodeInfo& operator=(NodeInfo&& other) MAIDSAFE_NOEXCEPT {
     id = std::move(other.id);
     dht_fob = std::move(other.dht_fob);
+    connected = std::move(other.connected);
```

Comment:
Yeah - I agree that it's overkill and that copying is equivalent, but I'm still very wary of the liberal scattering of unconditional `noexcept`s we have, so if this gets refactored to not be a POD, or we add a new member and copy the initalisation list code to accidentally copy the new member rather than move it then we'll end up in trouble.